orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -72,21 +72,29 @@ class MonitorKatoTask implements RetryableTask {
     }
 
     Task katoTask
+    def firstNotFoundRetrySkippingReplica = stage.context."kato.task.firstNotFoundRetry.skipReplica" as Long
```

Comment:
Including the `firstNotFoundRetry` in this key seems strange ... 

`def skipReplica = stage.context."kato.task.skipReplica" ?: false` instead?

I don't really think it needs to be a timestamp?

Reply:
I had something else in mind using the stamp, but since we are only retrying the master once, I like the idea of using a boolean